Dockerfile

```diff
@@ -147,6 +147,13 @@ RUN wget -P /build/software/go https://dl.google.com/go/go1.10.linux-amd64.tar.g
 RUN wget -P /build/software/go https://dl.google.com/go/go1.12.5.linux-amd64.tar.gz \
     && tar -xzf /build/software/go/go1.12.5.linux-amd64.tar.gz --directory /build/software/go && mv /build/software/go/go /build/software/go/go-1.12.5 && rm /build/software/go/go1.12.5.linux-amd64.tar.gz
 
+RUN cp -r /build/software/go/go-1.12.5 /usr/local/go
+
+ENV GOPATH /go
```

Comment:
Every path used for building needs to be located under /build.

Reply:
Change the path to /build/software/go/go